### PR TITLE
[spring-framework] Intial Integration for jdbc

### PR DESCRIPTION
**Pull request by aschaich**
Also has some commits to clean up the Docker/build.sh in preparation

### CODE REVIEW

File: projects/spring-framework/JdbcCoreMapperFuzzer.java [hunk 1]
```diff
@@ -0,0 +1,111 @@
+import com.code_intelligence.jazzer.api.FuzzedDataProvider;
```
Comment (Navidem) [inline projects/spring-framework/JdbcCoreMapperFuzzer.java:1]: Please add Copyright header.

File: projects/spring-framework/jdbc/MapperFuzzerServer.java [hunk 2]
```diff
@@ -0,0 +1,27 @@
+package jdbc;
```
Comment (Navidem) [inline projects/spring-framework/jdbc/MapperFuzzerServer.java:1]: Missing Copyright here and following .java files

File: projects/spring-framework/JdbcCoreMapperFuzzer.java (hunk 1 above)
Comment (aschaich) [reply projects/spring-framework/JdbcCoreMapperFuzzer.java:17]: Added in https://github.com/google/oss-fuzz/pull/8359/commits/3fee0cfc8fd8ee7f05701fd86ae1a717d7324c03

File: projects/spring-framework/jdbc/MapperFuzzerServer.java (hunk 2 above)
Comment (aschaich) [reply projects/spring-framework/jdbc/MapperFuzzerServer.java:16]: Added in https://github.com/google/oss-fuzz/pull/8359/commits/3fee0cfc8fd8ee7f05701fd86ae1a717d7324c03

**Review by Navidem — APPROVED** [review]
LGTM

**Pull request merged**